packages/framework/tsup.config.ts

```diff
@@ -1,18 +1,22 @@
-import { defineConfig } from 'tsup';
-import { type SupportedFrameworkName } from './src';
+import { type Options, defineConfig } from 'tsup';
+import { version } from './package.json';
+import { type SupportedFrameworkName } from './src/internal';
 
 const frameworks: SupportedFrameworkName[] = ['h3', 'express', 'next', 'nuxt', 'sveltekit', 'remix', 'lambda', 'nest'];
 
 const baseConfig: Options = {
   entry: ['src/index.ts', 'src/internal/index.ts', ...frameworks.map((framework) => `src/servers/${framework}.ts`)],
   sourcemap: false,
   clean: true,
-  treeshake: true,
   dts: true,
   minify: true,
   minifyWhitespace: true,
   minifyIdentifiers: true,
   minifySyntax: true,
+  define: {
+    SDK_VERSION: `"${version}"`,
```

Comment:
Static code replacements allow us to eliminate importing `package.json` on runtime that can cause some CJS/ESM issues.

Reply:
Awesome! 👍